tasks/install-matlab/v1/task.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+    "id": "553fa7ff-af12-4821-8ace-6bf3dc410e62",
+    "name": "InstallMATLAB",
+    "friendlyName": "Install MATLAB",
+    "description": "[PREVIEW] Install MATLAB on a Microsoft-hosted agent. Public licensing is not available for transformation products, such as MATLAB Coder and MATLAB Compiler.",
+    "helpMarkDown": "",
+    "category": "Tool",
+    "author": "The MathWorks, Inc.",
+    "preview": true,
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "inputs": [
+        {
+            "name": "release",
+            "type": "string",
+            "label": "Release",
+            "required": false,
+            "defaultValue": "latest",
+            "helpMarkDown": "MATLAB release to install. You can specify R2020a or a later release. If you do not specify `release`, the task installs the latest release of MATLAB."
```

Comment:
What happens if someone specifies a release before R2020a? Seems like mpm would still download it and then it would fail to license?

Reply:
Yeah matlab-batch will not work. I suppose this could still be OK for on-prem workflows that aren't using token licensing, but I'm not sure if we'd like to advertise that separately.

Reply:
It would go to matlab-batch and fall back to looking for traditional licensing? If so, that sounds good.

Reply:
Right, in the run-matlab-command logic, it just defaults to traditional licensing if the MATLAB at the top of path has a version is outside of the matlab-batch range.